pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -112,6 +112,10 @@ func convertService(svc v1.Service, domainSuffix string) *model.Service {
 		LoadBalancingDisabled: loadBalancingDisabled,
 		MeshExternal:          meshExternal,
 		Resolution:            resolution,
+		Namespace:             svc.Namespace,
+		Name:                  svc.Name,
+		CreationTimestamp:     svc.CreationTimestamp,
+		ResourceVersion:       svc.ResourceVersion,
```

Comment:
Debugging. Any platform we support can have a CreationTimestamp.

I can rename Namespace/Name to 'ConfigLocation' or similar, and remove ResourceVersion if you feel it's too k8s specific. And I agree, the intent was not that.

What we really need is:
- a way to tell user where the config that breaks pilot is coming from
- have a timestamp - file or most other registries support that, and if they don't we can set to 0.

The timestamp in turn can provide an indication of ordering - so in case of conflict we can drop newest configs. Or just provide the info for the user.

Happy to rename - the intent is certainly not to add k8s-specific stuff.

Reply:
Okay. In that case drop name and namespace as hostname already has this in fqdn form. All you need is creationtimestamp. And then in model.Services() Call, sort the array by creation time. That should be sufficient to ignore conflicting new services. We can still display useful info (name and namespace) as the fqdn has it. Service entries are global (no namespace). So in all cases, hostname and time stamp should suffice.